account_invoice_triple_discount/migrations/16.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from openupgradelib import openupgrade
+
+
+def migrate_discount_to_discount1(env):
```

Comment:
You should take part of this function in the install hook

Reply:
can you check again?